templates/components/cart/content.html

```diff
@@ -133,7 +133,6 @@ <h4 class="cart-item-name"><a href="{{url}}">{{name}}</a></h4>
                                 <svg><use xlink:href="#icon-close"></use></svg>
                             </a>
                         {{/if}}
-                        <span class="cart-remove cart-remove--spacer"></span>
```

Comment:
not sure why that is there but it creates an extra space which misaligns with the other items